### ISSUE TITLE
Minecraft wasn't removed.

### ISSUE THREAD

**Issue by toddf2p** [Sycnex/Windows10Debloater#57]
Downloaded fresh copy of 1803 ISO via Windows Media Creation tool, put on usb, installed to system, ran all updates, ran powershell ISE as admin, pasted Windows10Debloater.ps1 content into ISE. Chose Yes to remove all bloat. 

Minecraft was still left over. Simply right clicked and uninstalled. 

Based on the readme I assume Minecraft is supposed to be uninstalled as part of Windows10Debloater.ps1

Unrelated, is it possible to change Open File Explorer to This PC instead of Quick Access via powershell? Same thing, but to remove people/inkspace/set search to hidden?

**Comment by Sycnex** [comment]
Yes it is supposed to be removed. However, I have found that the leftover tiles on the start menu are just tiles and the application itself is no longer installed.

And yes there certainly is a way to make it open to This PC via windows settings but I can make it do that via PowerShell. Here you go: https://www.thewindowsclub.com/make-explorer-open-this-pc-quick-access-windows-10

I'm not entirely sure if you can actively hide those but I can take a look.

**Comment by kb9zzo** [comment]
I'm experiencing the same issue with Minecraft, Candy Crush Saga and Microsoft Solitare Collection.  Because the tiles are still there, all the user has to do is to click on the tile and then it installs and lets the user play.  When we do a Get-AppxPackage and a Get-AppxProvisionedPackage, they aren't listed, but the tiles are still there.  After the user clicks and installs the app, then it gets listed, but then we have to manually run the Remove-AppxPackage to get rid of it.  Is there anyway to remove the tile as well?

**Comment by Sycnex** [comment]
I haven't found a reliable way to remove the tiles but I will certainly look into it.

**Comment by fredenocs** [comment]
I have to comment on this, as I thought it wasn't working properly either.

I've imaged 100 machines in the last week or so.  Using MDT and sysprep version runs part of the process.  Creates a default administrator account, fine-whatever.  Tiles remain as if the apps are still present.

In my testing of my deployments, I've noticed the tiles are gone and cleaned when a new user is created, no matter how many users I create.  Administrator account remains with the tiles.  I didn't test the theory it re-installs the Apps, since I don't care too

If you want those tiles gone, then create a custom start menu and taskbar to load when a user logs in.
